### PR TITLE
Update everything to latest versions

### PR DESCRIPTION
**Pull request by scottschmitz**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,53 +1,105 @@
...
+        version_patch = 1
+        version_name = "$version_major.$version_minor.$version_patch"
+
+        activity_compose_version = "1.3.0-rc02"
+        app_compat_version = "1.3.0"
+        compose_version = '1.0.2'
+        compose_navigation_version = "2.4.0-alpha04"
+        core_ktx_version = "1.6.0"
+        detekt_version = '1.15.0-RC1'
+        kotlin_version = '1.5.10'
```
Comment (joshing-dev) [inline build.gradle]: I had to update this to 1.5.21 in order to get it to build

Comment (scottschmitz) [reply build.gradle]: updated

**Review by allisonjoycarter — APPROVED** [review]
Runs for me 🤷🏻‍♀️

**Comment by scottschmitz** [comment]
@Matrix159 - I'm going to merge because I think it's required for anyone else to do anything. If you're still having issues we can try to debug on your machine.

**Pull request merged**